packages/next/README.md

```diff
@@ -1 +1,71 @@
-# canon-next
+# canon-next: Nextjs components for sites migrating away from canon-core.
+
+This package provides React Components for rendering a canon profile on a NextJS app, using Mantine components.
+
+# Setup an installation
+
+You'll need to create a NextJS (v13) app. Install the package using npm:
+
+```npm i @datawheel/canon-next```
+
+You'll need to wrap your entire app into a `MantineProvider`, in order to allow Mantine components to render properly. For detailed instructions follow the official [Mantine Documentation for NextJS](https://mantine.dev/guides/next/). In adition, you'll need to set up `next-i18next` on your app, with the appropiate locales.
```

Comment:
spelling: "adition" to "addition"

Also, could you put a very simple example here of what `_app.js` would look like on initial install, with the providers and translations set up?

Reply:
added also a link to our template, that has all already installed